c++/src/capnp/dynamic-test.c++

```diff
@@ -473,6 +473,40 @@ TEST(DynamicApi, SetDataFromText) {
   EXPECT_EQ(data("foo"), root.get("dataField").as<Data>());
 }
 
+void wrappedEnumListTest(std::string field)
+{
+  MallocMessageBuilder builder;
+  auto root = builder.initRoot<DynamicStruct>(Schema::from<TestAllTypes>());
+  auto wrapper = root.init(field).as<DynamicStruct>();
+  wrapper.set("value", {TestEnum::BAR, TestEnum::FOO});
+  checkList<TestEnum>(wrapper.get("value"), {TestEnum::BAR, TestEnum::FOO});
+}
+
+TEST(DynamicApi, SetWrappedEnumListFromNative) {
+  wrappedEnumListTest("wrappedEnumList");
+}
+
+TEST(DynamicApi, SetGenericWrappedEnumListFromNative) {
+  wrappedEnumListTest("genericWrappedEnumList");
```

Comment:
This test fails due to a null schema ptr when calling Type::asEnum().
The rest of the tests work correctly.